templates/nextroute/main.go

```diff
@@ -36,6 +37,15 @@ func solver(
 		return runSchema.Output{}, err
 	}
 
+	if options.SanityCheck.Enable {
```

Comment:
Why do we need to add this to the template? Can't we check if the option is enabled on the factory side in platform and if it is then add the sanity check to the output?

Reply:
We do not **add** the  sanity check but enable it excluding solving. The sanity check can take a lot of time (away from solving).